todo_app/flask_config.py

```diff
@@ -2,7 +2,16 @@
 
 
 class Config:
-    """Base configuration variables."""
-    SECRET_KEY = os.environ.get('SECRET_KEY')
-    if not SECRET_KEY:
-        raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
+    def __init__(self):
+        """Base configuration variables."""
+        SECRET_KEY = os.environ.get('SECRET_KEY')
```

Comment:
This should set `self.SECRET_KEY = ...`. It's currently just setting a local variable inside this function that then gets thrown away.

It doesn't matter if you're not currently using the Flask session, but you will need it again later in the course